### PR TITLE
[DOC-9712] Add CIDR details for GCP

### PR DESCRIPTION
**Pull request by mdlinville**
[DOC-9712] Add CIDR details for GCP

### CODE REVIEW

File: src/current/cockroachcloud/create-an-advanced-cluster.md [hunk 1]
```diff
@@ -58,7 +60,7 @@ You can use [VPC peering]({% link cockroachcloud/network-authorization.md %}#vpc
 
 VPC peering is available only for GCP clusters. For clusters deployed on AWS, you can [configure AWS PrivateLink]({% link cockroachcloud/network-authorization.md %}#aws-privatelink) after creating your cluster. [Azure Virtual Network Peering](https://learn.microsoft.com/azure/virtual-network/virtual-network-peering-overview) is not yet supported. Refer to [CockroachDB {{ site.data.products.advanced }} on Azure]({% link cockroachcloud/cockroachdb-advanced-on-azure.md %}).
 
-You can use CockroachDB {{ site.data.products.cloud }}'s default IP range and size (`172.28.0.0/14`) as long as it doesn't overlap with the IP ranges in your network. Alternatively, you can configure the IP range:
+You can use CockroachDB {{ site.data.products.cloud }}'s default IP range and size (`172.28.0.0/14`) as long as it doesn't overlap with the IP ranges in your network. Alternatively, you can configure the IP range. If you add regions later, the IP range will be automatically populated for added regions.
```
Comment (udnay) [inline src/current/cockroachcloud/create-an-advanced-cluster.md:63]: > If you add regions later, the IP range will be automatically populated for added regions.

Maybe something like:
If you add regions later, the `/19` IP range for the new region will be taken from the CIDR range configured at cluster creation.

File: src/current/cockroachcloud/advanced-cluster-management.md [hunk 2]
```diff
@@ -98,18 +98,18 @@ You can add up to nine regions at a time through the Console. See [Planning your
 
 1. On the **Regions** page, click **Add a region**.
 
-    If you have a GCP cluster with [VPC peering]({% link cockroachcloud/network-authorization.md %}) enabled, the IP range will be automatically populated for added regions.
+    If you have a GCP cluster with [VPC peering]({% link cockroachcloud/network-authorization.md %}) enabled, the IP range will be automatically populated for added regions. For GCP, each region requires a `/19` CIDR block.
```
Comment (udnay) [inline src/current/cockroachcloud/advanced-cluster-management.md:101]: Same comment as below, making it clear that we configure the `/19` CIDR block out of the initially configured CIDR block

File: src/current/cockroachcloud/create-an-advanced-cluster.md (hunk 1 above)
Comment (mdlinville) [reply src/current/cockroachcloud/create-an-advanced-cluster.md]: Done, PTAL

File: src/current/cockroachcloud/advanced-cluster-management.md (hunk 2 above)
Comment (mdlinville) [reply src/current/cockroachcloud/advanced-cluster-management.md]: Done, PTAL

File: src/current/cockroachcloud/advanced-cluster-management.md [hunk 3]
```diff
@@ -98,18 +98,18 @@ You can add up to nine regions at a time through the Console. See [Planning your
 
 1. On the **Regions** page, click **Add a region**.
 
-    If you have a GCP cluster with [VPC peering]({% link cockroachcloud/network-authorization.md %}) enabled, the IP range will be automatically populated for added regions.
+    For GCP, Each region consumes a `/19` CIDR range from the CIDR range configured during cluster creation. If you add a region a GCP cluster with [VPC peering]({% link cockroachcloud/network-authorization.md %}) enabled, the region's IP range will be automatically assigned from the cluster's CIDR range.
```
Comment (udnay) [inline src/current/cockroachcloud/advanced-cluster-management.md:101]: > If you add a region `to` a GCP

Missing a word I think.

/nit
> the region's IP range will be automatically assigned from the cluster's CIDR range.

This work flow is true if they have VPC peering enabled or not

Comment (mdlinville) [inline src/current/cockroachcloud/advanced-cluster-management.md:101]:
```suggestion
    For GCP, each region consumes a `/19` CIDR range from the CIDR range configured during cluster creation. If you add a region to a GCP cluster with [VPC peering]({% link cockroachcloud/network-authorization.md %}) enabled, the region's IP range will be populated automatically.
```

Comment (mdlinville) [reply src/current/cockroachcloud/advanced-cluster-management.md:101]: Thanks, I was trying to copy some of the detail from the cluster-create page, which says it's populated automatically if you add additional regions. Does this work?

**Pull request merged**